src/sidebar/util/test/observer-element-size-test.js

```diff
@@ -0,0 +1,100 @@
+'use strict';
+
+const observeElementSize = require('../observe-element-size');
+
+/**
+ * Give MutationObserver, ResizeObserver etc. a chance to deliver their
+ * notifications.
+ */
+function waitForObservations() {
+  return new Promise(resolve => setTimeout(resolve, 1));
+}
+
+describe('observeElementSize', () => {
+  let content;
+  let sizeChanged;
+  let stopObserving;
+
+  beforeEach(() => {
+    sizeChanged = sinon.stub();
+    content = document.createElement('div');
+    content.innerHTML = '<p>Some test content</p>';
+    document.body.appendChild(content);
+  });
+
+  afterEach(() => {
+    stopObserving();
+    content.remove();
+  });
+
+  function startObserving() {
+    stopObserving = observeElementSize(content, sizeChanged);
+  }
+
+  context('when `ResizeObserver` is available', function() {
+    if (typeof ResizeObserver === 'undefined') {
```

Comment:
Can you help me understand this branch? When would we expect this condition to be truthy? When tests are being run in unexpected environments? And are there potential risks to this set of tests being skipped and/or us not noticing?

Reply:
By default tests run in headless Chrome where `ResizeObserver` is available. However you can manually run tests in any other browser locally by opening the Karma URL (printed when tests start running) in that browser. The only way this will fail is if some other test were to explicitly overwrite this global. In that case though the test runner would notify the user that some tests have been skipped.